test/lib/jdk/test/whitebox/code/Compiler.java

```diff
@@ -88,6 +88,25 @@ public static boolean isGraalEnabled() {
         return true;
     }
 
+    /**
+     * Check if libgraal is used as JIT compiler.
+     *
+     * libraal is enabled if isGraalEnabled is true and:
+     * - UseJVMCINativeLibrary flag is true
+     *
+     * @return true if libgraal is used as JIT compiler.
+     */
+    public static boolean isLibgraalEnabled() {
+        if (!isGraalEnabled()) {
+            return false;
+        }
+        Boolean useJvmciNativeLibrary = WB.getBooleanVMFlag("UseJVMCICompiler");
```

Comment:
Typo: "UseJVMCICompiler". Should be "UseJVMCINativeLibrary".

Reply:
That's embarrassing! Thanks for catching it.